surveys/2021-trademark-survey/questions.md

```diff
@@ -148,7 +150,7 @@ Pick as many as you agree with:
 >
 > TODO
 
-### Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have made a financial contribution to the Rust Foundation.
+### Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust.
```

Comment:
This change definitely makes the question different than the original intent. Not choosing sides on which is better or not, but I can actually see these potentially being two separate questions.

Reply:
That's true. We could ask the other question ("provided they have an open source project that uses Rust"), but that seems like a much broader ask likely to get different answers. (And I was trying to avoid increasing the number of questions.)

Reply:
Also, for context, note that GitHub's diff here is comparing to the wrong question here. The actual change is from "Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have an open source project that uses Rust" to "Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust"

Reply:
I think we're missing an "only" or "provided":

> Commercial businesses can use the Rust/Cargo logo on their website or social media site, *only* in the context of an open source project that uses Rust.